bitnami/aspnet-core/templates/deployment.yaml

```
@@ -114,12 +114,16 @@ spec:
           {{- if .Values.lifecycleHooks }}
           lifecycle: {{- include "common.tplvalues.render" (dict "value" .Values.lifecycleHooks "context" $) | nindent 12 }}
           {{- end }}
+          {{- if .Values.appFromExternalRepo.enabled }}
           workingDir: /app
+          {{- end }}
           command:
           {{- if .Values.command }}
           {{- include "common.tplvalues.render" (dict "value" .Values.command "context" $) | nindent 12 }}
           {{- else if .Values.appFromExternalRepo.enabled }}
           {{- include "common.tplvalues.render" (dict "value" .Values.appFromExternalRepo.startCommand "context" $) | nindent 12 }}
+          {{- else if .Values.useContainerCommand }}
+            {{- /* left empty if use container command */ -}}
```

@yip102011 Correct me if I'm missing something but as it is now we'll just need to use a custom `command` and `vars` to replicate your Dockerfile command right? I agree it can be a minor nuisance caused by the chart not using the container's default, but I don't think this justifies adding a new parameter.

A middle point would be to differentiate between `command=null` and `!=null`. We could add the following logic to avoid setting anything when using `null`:

```suggestion
          {{- if .Values.command }}
          {{- include "common.tplvalues.render" (dict "value" .Values.command "context" $) | nindent 12 }}
          {{- else if kindIs "invalid" .Values.command }}
          {{- /* will use container container */ -}}
          {{- else if .Values.appFromExternalRepo.enabled }}
          {{- include "common.tplvalues.render" (dict "value" .Values.appFromExternalRepo.startCommand "context" $) | nindent 12 }}
```

wdyt?